code/game/objects/items/cards_ids.dm

```diff
@@ -5,7 +5,7 @@
 #define ID_ICON_BORDERS 1, 9, 32, 24
 
 /// Fallback time if none of the config entries are set for USE_LOW_LIVING_HOUR_INTERN
-#define INTERN_THRESHOLD_FALLBACK_HOURS 15
+#define INTERN_THRESHOLD_FALLBACK_HOURS 15 HOURS
```

Comment:
This is no longer in "hours" units and should probably be named something like `INTERN_THRESHOLD_FALLBACK_TIME`